### DISCUSSION TITLE
TypeScript

### DISCUSSION THREAD

**Discussion by RamazanYilmazz** [OmerAtayilmaz/learning-typescript#11] [Q&A] [answered]
What Exactly Are TypeScript's Components? can you open it a little more?

**Comment by OmerAtayilmaz** [comment, marked as answer]
> What Exactly Are TypeScript's Components? can you open it a little more?

Yup, Thing Components as a structure. 
For exampe, lets parse a website to components.
You can create a NavComponent that includes Navbar links or you can create A component as "Content" that includes more component in itself.

Here is the more example about component;

1. BlogComponent
2. PostComponent
3. PostListComponent
4. SidebarComponent
5. PostDetailComponent
6. FooterComponent

as you see at the above, we can parse the website like puzzle :+1: 

That is exactly what is component

**Reply by RamazanYilmazz** [reply]
thanksssssss
